### DISCUSSION TITLE
Merci !!

### DISCUSSION THREAD

**Discussion by zelerkill** [codecrafting-io/asus-setup-tool#4] [General]
Désolé mon commentaire sera en Français.
Merci, merci d'avoir créé cet outil. Le support Asus était incapable de résoudre mon problème, et votre outil la fait parfaitement. J'ai regardé beaucoup de forum avant de tomber sur un de vos commentaires. je pensais même ne pas réactiver. Vous avez sauvé mes leds ^^

**Comment by codecrafting-io** [comment]
Salut, merci. Désolé pour mon français (j'ai utilisé un traducteur pour m'aider). Par curiosité, quelle version utilisez-vous ? Et si l'outil disposait d'un support en français, cela serait-il plus facile pour vous ?

**Comment by zelerkill** [comment]
Hello,If you prefer, I can try to speak inglish.I took the latest version (v0.6.0) and and used the chrome translation to follow the information you provided.Personally, it didn't give me any problems. If it helps, I'm on Windows 11 and I've installed the old version of aura, and AISuite3. And everything went fine.
Thanks again !
